cmd/kubeadm/app/cmd/reset_test.go

```diff
@@ -247,52 +245,15 @@ func newFakeDockerChecker(warnings, errors []error) preflight.Checker {
 	return &fakeDockerChecker{warnings: warnings, errors: errors}
 }
 
-func TestResetWithDocker(t *testing.T) {
-	fcmd := fakeexec.FakeCmd{
-		RunScript: []fakeexec.FakeRunAction{
-			func() ([]byte, []byte, error) { return nil, nil, nil },
-			func() ([]byte, []byte, error) { return nil, nil, errors.New("docker error") },
-			func() ([]byte, []byte, error) { return nil, nil, nil },
-		},
-	}
-	fexec := fakeexec.FakeExec{
-		CommandScript: []fakeexec.FakeCommandAction{
-			func(cmd string, args ...string) exec.Cmd { return fakeexec.InitFakeCmd(&fcmd, cmd, args...) },
-			func(cmd string, args ...string) exec.Cmd { return fakeexec.InitFakeCmd(&fcmd, cmd, args...) },
-			func(cmd string, args ...string) exec.Cmd { return fakeexec.InitFakeCmd(&fcmd, cmd, args...) },
-		},
-	}
-	resetWithDocker(&fexec, newFakeDockerChecker(nil, nil))
-	if fcmd.RunCalls != 1 {
-		t.Errorf("expected 1 call to Run, got %d", fcmd.RunCalls)
-	}
-	resetWithDocker(&fexec, newFakeDockerChecker(nil, nil))
-	if fcmd.RunCalls != 2 {
-		t.Errorf("expected 2 calls to Run, got %d", fcmd.RunCalls)
-	}
-	resetWithDocker(&fexec, newFakeDockerChecker(nil, []error{errors.New("test error")}))
-	if fcmd.RunCalls != 2 {
-		t.Errorf("expected 2 calls to Run, got %d", fcmd.RunCalls)
-	}
-}
-
-func TestResetWithCrictl(t *testing.T) {
+func TestRemoveContainers(t *testing.T) {
```

Comment:
improve this unit test with more test cases?

Reply:
done. 100% coverage